sdk/appconfiguration/src/feature_manager/mod.rs

```diff
@@ -0,0 +1,172 @@
+use azure_core::{auth::TokenCredential, FixedRetryOptions, RetryOptions};
+use std::{collections::HashMap, fs::File, sync::Arc};
+
+use self::{
+    app_context::ContextHolder, feature_filter::FeatureFilter, feature_holder::FeatureHolder,
+    feature_type::Feature,
+};
+
+pub mod app_context;
+mod feature_filter;
+mod feature_holder;
+mod feature_type;
+mod models;
+
+pub const FEATURE_PREFIX: &str = ".appconfig.featureflag/";
+
+#[derive(Clone)]
+pub struct FeatureExplorer {
+    holder: Arc<FeatureHolder>,
+    context: Option<Arc<dyn ContextHolder>>,
+    // use for dev perspective, to override the feature value from the remote
+    on_off: HashMap<String, Vec<Feature>>,
+    client: azure_svc_appconfiguration::Client,
+}
+
+#[derive(Clone)]
+pub struct FeatureExplorerBuider {
+    endpoint: Option<String>,
+    credential: Arc<dyn TokenCredential>,
+    context: Option<Arc<dyn ContextHolder>>,
+    on_off: HashMap<String, Vec<Feature>>,
+    retry: Option<RetryOptions>,
+}
+
+impl FeatureExplorerBuider {
```

Comment:
I believe this is a typo, this should be FeatureExplorerBuilder?

Reply:
done, thanks!